libs/components/ag-grid/src/lib/modules/ag-grid/types/cell-type.ts

```diff
@@ -40,6 +40,15 @@ export enum SkyCellType {
    * Specifies that cells in the column will display the currently selected date formatted as `MM-DD-YYYY`, or the date format of the locale passed to `getGridOptions()`. If you would like to overwrite this format, you can [define a `valueFormatter`](https://www.ag-grid.com/javascript-grid-value-formatters/) on the column definition. See the demo for an example.
    */
   Date = 'skyCellDate',
+  /**
+   * **Edit mode**
+   * <br/>
+   * Specifies that cells in the column will be edited as  [SKY UX lookup components](https://developer.blackbaud.com/skyux-v5/components/lookup). You can set any of the lookup component's properties by passing them in the [column definition's `cellEditorParams` property](https://www.ag-grid.com/javascript-grid-column-properties/). These params can be updated as other cell edits are made or [provided dynamically](https://www.ag-grid.com/javascript-grid-cell-editing/#dynamic-parameters) based on other cell values. See the demo for an example. Text can be entered and a value selected from the provided list.
+   * <br/><br/>
+   * **Read-only mode**
+   * <br/>
+   * Specifies that cells the column will display, by default, either: the name(s) of the selected value(s) if there are less than 6, or a summary count of the values if there are more than 5. If the lookup needs to show a different property or needs to be formatted in any way, you can [define a `valueFormatter`](https://www.ag-grid.com/javascript-grid-value-formatters/) on the column definition.
```

Comment:
Maybe should have "if there are less than 6 selected values"? Probably worth having @blackbaud-johnly double check verbiage.

Reply:
```suggestion
   * Specifies that cells in the column will either display the names of up to five selected values or a summary count if there are more than 5. If the lookup needs to show a different property or needs to be formatted in any way, you can [define a `valueFormatter`](https://www.ag-grid.com/javascript-grid-value-formatters/) on the column definition.
```